### PR TITLE
Polish doc

### PR DESCRIPTION
**Pull request by izeye**
This PR polishes doc a bit.

### CODE REVIEW

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc [hunk 1]
```diff
@@ -1849,7 +1849,7 @@ The following cache libraries are supported:
 Metrics are tagged by the name of the cache and by the name of the `CacheManager` that is derived from the bean name.
 
 NOTE: Only caches that are configured on startup are bound to the registry.
-For caches not defined in the cache’s configuration, that is caches created on-the-fly or programmatically after the startup phase, an explicit registration is required.
+For caches not defined in the cache’s configuration, that are created on-the-fly or programmatically after the startup phase, an explicit registration is required.
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: I really meant "that is" as "i.e.". Perhaps there's a missing comma somewhere?

Comment (izeye) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: @snicoll I'm not an English native speaker, either :sweat_smile: I just thought that the "that" clause was meant to be for the "caches" as "that is caches" sounded unusual to me. I might be wrong here.

Comment (snicoll) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: OK. I am not sure either :) Would you be happy with

> For caches not defined in the cache’s configuration, i.e. caches created on-the-fly or programmatically after the startup phase, ...

Comment (izeye) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: @snicoll Sounds better to me 😄

Comment (snicoll) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: Sweet, would you be willing to update your proposal with that please?

Comment (izeye) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/production-ready-features.adoc]: @snicoll Sure. I've done that in 44156e8795f0588ebe3d522e64822ca9a1aa6e2d.

**Pull request closed without merging**